code-challenges.js

```diff
@@ -21,48 +21,97 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
+// Start by creating a function called checkBoil with a parameter of userTemp.
+// Next, use a series of conditionals to first check if the input arguement is above boiling point, returning if it is.
+// Then, we use an else if statement to do the same for at and below boiling point, and a final catch-all else for bad inputs.
+
+const checkBoil = (userTemp) => {
+    const boilingPoint = 212
```

Comment:
Awesome! Love that you used a variable here and used it in conditional statement. You can include that in your pseudocode to help make it more descriptive and paint a complete picture.